projects/github-actions/repo-gardening/src/tasks/gather-support-references/index.js

```diff
@@ -264,7 +264,11 @@ async function addOrUpdateInteractionCountLabel(
 	number,
 	issueReferencesCount
 ) {
-	const ranges = [ 50, 20, 10 ];
+	let ranges = [ 50, 20, 10 ];
+
+	if ( 'Automattic/sensei' === repo ) {
+		ranges = [ 5 ];
```

Comment:
Noting this will set it so there's only one band (e.g. more than 5) instead of the under 10, 10-20, 20-50, etc).